client/src/components/pages/application/Accounts/index.js

```diff
@@ -0,0 +1,184 @@
+import { useHistory } from 'react-router-dom';
+import { Helmet } from 'react-helmet';
+
+import TextField from '../../../common/TextField';
+import Typography from '../../../common/Typography';
+import backGround from '../../../../assets/images/backGround.svg';
+import gitHub from '../../../../assets/icons/github.svg';
+import discord from '../../../../assets/icons/discord.svg';
+import freeCodeCamp from '../../../../assets/icons/freeCodeCamp.png';
+import codeWars from '../../../../assets/icons/codewars-512.png';
+import Button from '../../../common/Button';
+import Limitation from '../../../common/limitation';
+import AccountsVallation from '../../../../utils/application/AccountsValidation';
+
+import './index.css';
+
+const getUserID = async () => {
+  const { data } = await axios.get('/api/v1/isUser');
+  return data;
+};
+
+const Accounts = () => {
+  const [githubLink, setGitHub] = useState('');
+  const [freeCodeCampLink, setFreeCode] = useState('');
+  const [codeWarsLink, setCodeWars] = useState('');
+  const [joinDiscord, setDiscord] = useState(false);
+  const [message, setMessage] = useState([]);
+  const [UserId, setId] = useState('');
+  const [userName, setName] = useState('');
+  const history = useHistory();
+
+  const throwMessage = (msg) => setMessage(msg);
+  useEffect(() => {
+    getUserID().then((data) => {
+      if (data.message === 'you are authorized') {
+        setId(data.userId);
+      }
+    });
+    if (UserId) {
+      axios.get(`/api/v1/applicants/${UserId}`).then(({ data: { user } }) => {
+        setGitHub(user.githubLink);
+        setFreeCode(user.freeCodeCampLink);
+        setCodeWars(user.codeWarsLink);
+        setName(user.fullName);
+      });
+    }
+  }, [UserId]);
+  const Next = () => {
+    AccountsVallation({ githubLink, freeCodeCampLink, codeWarsLink })
+      .then(() =>
+        axios.patch(`/api/v1/applicants/${UserId}`, {
+          githubLink,
+          freeCodeCampLink,
+          codeWarsLink,
+          joinDiscord,
+        })
+      )
+      .then(() => history.push('/tasks'))
+      .catch(({ errors }) => throwMessage(errors));
+  };
+
+  const Name = userName.split(' ')[0];
+  return (
+    <div className="Container_page">
+      <Helmet>
+        <title>Accounts</title>
+      </Helmet>
+      <img src={backGround} alt="backGround" className="backGround" />
+      {UserId && userName ? (
+        <>
+          <div className="text_Welcome">
+            <Typography variant="h3" color="default">
+              Welcome, {Name}
+            </Typography>
+          </div>
+          <div className="Form_container">
+            <Typography variant="h5">Accounts</Typography>
+            <>
+              <div className="label_container">
+                <img src={gitHub} alt="GitHub" />
+                <Typography className="label">Github</Typography>
+              </div>
+              <TextField
+                name="githubLink"
+                value={githubLink}
```

Comment:
there is a problem in displaying githublink in the value, but we want to set the value to empty string and give the applicant the link example as placeholder